plugins/TelescopeControl/src/gui/TelescopeConfigurationDialog.cpp

```diff
@@ -155,7 +155,12 @@ void TelescopeConfigurationDialog::createDialogContent()
 	connect(ui->radioButtonTelescopeRTS2, SIGNAL(toggled(bool)), this, SLOT(toggleTypeRTS2(bool)));
 	connect(ui->radioButtonTelescopeINDI, SIGNAL(toggled(bool)), this, SLOT(toggleTypeINDI(bool)));
 	#ifdef Q_OS_WIN
-	connect(ui->radioButtonTelescopeASCOM, SIGNAL(toggled(bool)), this, SLOT(toggleTypeASCOM(bool)));
+	#if (QT_VERSION>=QT_VERSION_CHECK(6,0,0))
+		ui->radioButtonTelescopeASCOM->setToolTip(q_("Disabled due to apparent incompatibility. Please use Qt5-based build."));
```

Comment:
"Use Qt5-based build" is a very cryptic advice for people far from software development. At the very least it should be "Use the Qt5-based build of Stellarium". Maybe even "Use the Qt5 build of Stellarium (the one for Windows 7+)" or something like this.

Reply:
How large shall tooltips get? Are hyperlinks to the download allowed?

Reply:
Hyperlinks will not work in tooltips

Reply:
The tooltips can have line breaks, which will reduce their horizontal size.

Reply:
There is always room for improvement...